### ISSUE TITLE
Add @conventionDeclarationReference, @maintenanceEventReference, and @sourceReference to EAD

### ISSUE THREAD

**Issue by kerstarno** [SAA-SDT/eas-schemas#11]
## Creator of issue

1. Kerstin Arnold
2. TS-EAS EAD team lead
3. @kerstarno
4.

## The issue relates to

- [ ] EAC-CPF schema issue
- [ ] EAC-CPF Tag Library issue
- [x] EAD schema issue
- [x] EAD Tag Library issue
- [ ] Schema issue
- [ ] Tag Library issue
- [ ] Suggestions for all schemas
- [ ] Suggestions for all Tag Libraries
- [ ] Other

## Wanted change/feature
* Text:  See https://github.com/SAA-SDT/TS-EAS-subteam-notes/blob/master/eaccpf-subteam/working-documents/topics/assertion/Topic_assertions_20200424.pdf

**Comment by kerstarno** [comment]
- [ ] **For assertion description add the following attributes as optional to all non-empty elements outside of `<control>` (i.e. exlcuding `<lb>` and `<ptr>`), including `<archdDesc>`, but not `<emph>` nor `<foreign>`**:
  - [ ] `@conventionDeclarationReference` with data type IDREFS
  - [ ] `@maintenanceEventDeclarationReference` with data type IDREFS
  - [ ] `@sourceDeclarationReference` with data type IDREFS

**_Note for working on and testing this issue: When the schema changes are done in development branch, please mark this by ticking the box for the higher level in the list (printed in bold); once the changes have been tested successfully please mark this by ticking the box._**

**Comment by fordmadox** [comment]
Either, this is ready for testing now in the 'development' branch.

**Comment by fordmadox** [comment]
Re-add those reference elements to "span", and remove "foreign" element altogether.

**Comment by kerstarno** [comment]
Tested with XSD and RNG. 

Currently, the following elements are missing the three @...Reference attributes:
- `<findAidDesc>` (though the relevant issue #27 hasn't been fully implemented yet in the draft schema)
- `<agent>` (with the extended use, see #58, `<agent>` will also be used outside of `<control>`)
- `<citedRange>` (with being a sub-element of `<findAidDesc>`, this also becomes an element outside of `<control>`)
- `<did>`
- `<controlAccess>` (to be integrated with the new element `<subjectHeadings>`, see #67)
- `<dsc>`
- `<container>`
- `<dao>` (to be integrated with the new element `<formAvailable>`, see #65)
- `<daoSet>` (same as `<dao>`)
- `<indexEntry>` (to be integrated with the new element `<subjectHeadings>`)
- `<c>`
- `<c01>` to `<c12>`
- `<span>` (as mentioned in the previous comment)
- `<foreign>` (to be removed, as mentioned in the previous comment)
- `<footnote>` (to be removed, see #70)
-  `<thead>` (to be removed, see #62)
- `<tgroup>` (same as `<thead>`)
- `<nameGrp>` (to be integrated with the new element `<subjectHeadings>`)
- `<row>` (same as `<thead>`)
- `<tbody>` (same as `<thead>`)
- `<entry>` (same as `<thead>`)

**Comment by kerstarno** [comment]
Re-tested with the XSD and the RNG with regard to the elements listed previously as missing the three attributes in question.

They have now been added to all of the above elements apart from `<dsc>` (now renamed to `<descriptionOfComponents>`). I have created a pull request with the respective change (#76). Once this pull request has been merged, this issue here will require one more round of testing.
